Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -137,6 +141,36 @@ The following command is listing shared libraries:
 $ otool -L <binary>
 ```
 
+#### Retrieving Strings
+
+Strings are always a good starting point while analyzing a binary, as it provides a context to the associated code. For instance, an error log string "Cryptogram generation failed." gives us a hint that the code might be responsible for generation of a cryptogram.
+
+Strings can be extracted from an iOS binary using a GUI tools like Ghidra or Cutter, using a CLI-based tool such as the *strings* Unix utility (`strings <path_to_binary>`) or radare2's rabin2 (`rabin2 -zz <path_to_binary>`). When using the CLI-based tools, you use them in conjunction with other tools like *grep* or *sort* to further filter and analyze the results.
```

Comment:
This paragraph comes from the Android part of the guide, if it's anyway the same let's leave it as it was.

```suggestion
In order to extract strings from an iOS binary, you can use GUI tools such as Ghidra or Cutter or rely on CLI-based tools such as the _strings_ Unix utility (`strings <path_to_binary>`) or radare2's rabin2 (`rabin2 -zz <path_to_binary>`). When using the CLI-based ones you can take advantage of other tools such as grep (e.g. in conjunction with regular expressions) to further filter and analyze the results.
```